README.md

```diff
@@ -49,16 +49,22 @@ A multi-cloud strategy is essential these days. But managed Kubernetes offerings
 With k0smotron, you can run the control plane management cluster in a public or private cloud provider of your choice and the worker nodes in various clouds. With this you get a homogenized and unified cluster management, providing one flavor on all clouds and saving the costs for the highly available Control Plane in the cloud.
 
 ## Getting Started
+
 Getting started with k0smotron is easy. Simply install the controller into an existing cluster:
 
 ```bash
 kubectl apply -f https://docs.k0smotron.io/stable/install.yaml
 ```
 
-## Creating a cluster
+You can also install k0smotron ClusterAPI providers via `clusterctl`:
+
+```shell
+clusterctl init --bootstrap k0smotron --control-plane k0smotron --config config.yaml
```

Comment:
I don't remember, why I haven't mentioned the infrastructure provider here, should we add `--infrastructure k0smotron` as well?

Reply:
I think that's good idea

Reply:
done